code/stable/dblpend/SRS/HTML/DblPend_SRS.html

```diff
@@ -2728,42 +2728,42 @@ <h1>Traceability Matrices and Graphs</h1>
               <td></td>
             </tr>
             <tr>
-              <td><a href=#inputValues>FR:Input-Values</a></td>
+              <td><a href=#correct>NFR:Correctness</a></td>
               <td></td>
               <td></td>
               <td></td>
               <td></td>
             </tr>
             <tr>
-              <td><a href=#verifyInptVals>FR:Verify-Input-Values</a></td>
+              <td><a href=#portable>NFR:Portability</a></td>
```

Comment:
These are just re-ordered. A tangential issue to this PR because it was previously dependant on the order of insertion rather than a specific ordering, unfortunately.